### DISCUSSION TITLE
v2.6.0-rc3

### DISCUSSION THREAD

**Discussion by mashb1t** [mashb1t/Fooocus#55] [Announcements]
## What's Changed
* chore: update python dependencies
* i18n: rename tab titles and translations from singular to plural
* i18n: rename document to documentation
* i18n: only import translator when needed

**Full Changelog**: https://github.com/mashb1t/Fooocus/compare/v2.6.0-rc2...v2.6.0-rc3

<hr /><em>This discussion was created from the release <a href='https://github.com/mashb1t/Fooocus/releases/tag/v2.6.0-rc3'>v2.6.0-rc3</a>.</em>

**Comment by bsdesigner** [comment]
Started - updated - but the version is still 2.6.0-rc2

![Знімок екрана 2024-07-17 003348](https://github.com/user-attachments/assets/69968cc3-3c38-4d61-82c9-7a4ee16dfe82)

**Reply by mashb1t** [reply]
@bsdesigner 2.6.0-rc3 is currently on develop and not on main (as it should be for RCs), so if you want to check it out you have to execute `git checkout v2.6.0-rc3`.

**Reply by bsdesigner** [reply]
I see! Sorry...

**Reply by mashb1t** [reply]
No worries, i put RCs by accident on main, which makes every user a beta user, and this is obviously not intended. Final releases for upstream and this repository will be released this week.

**Reply by bsdesigner** [reply]
Thank You!
